Library/Formula/libquantum.rb

```diff
@@ -0,0 +1,35 @@
+class Libquantum < Formula
+  desc "C library for the simulation of quantum mechanics"
+  homepage "http://www.libquantum.de/"
+  url "http://www.libquantum.de/files/libquantum-1.0.0.tar.gz"
```

Comment:
Thanks, could you add a short comment above the URL to explain that? This will prevent people from submitting PRs to update the formula to 1.1.1 (see [here for example](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/globus-toolkit.rb#L4)).